src/schemas/block.json

```
@@ -125,7 +125,8 @@
 	"BlockTag": {
 		"title": "Block tag",
 		"type": "string",
-		"enum": ["earliest", "latest", "pending"]
+		"enum": ["earliest", "latest", "pending", "finalized", "safe", "unsafe"],
+		"description": "Tags \"finalized\" and \"safe\" refer to the corresponding blocks defined by the most recent fork choice state. Tag \"unsafe\" is an alias to the \"latest\"."
```

Are newlines supported in the description string?  If not we could replace the `\n` below with `; ` or something.  I think there is value in defining each fairly clearly.
```suggestion
		"enum": ["earliest", "finalized", "safe", "unsafe", "pending"],
		"description": "`earliest`: The lowest numbered block the client has available.\n`finalized`: The most recent block that cannot be reorged outside of a hard fork.\n`safe`: The most recent block that is will not reorg unless there is an active ongoing attack.\n`unsafe`: The most recent block the client has seen that it believes will be part of the canonical chain (high probability of being reorged out).\n`pending`: A sample next block built by the client containing what it believes will be the set of transactions likely to show up in that block.\n`latest`: **DEPRECATED** Will be removed at some point in the future, currently an alias for `unsafe` or `safe`."
```
I also removed `latest` from the enum and only referenced it in the description to help make it clear that it is deprecated (but if people go looking for it they can find it).

---

If we remove `latest` it would read as it's already unaccessible. I can foresee users not finishing reading the description (thus, not seeing that it's been DEPRECATED but still available for a short period of time) and starting to ask why this tag that was in place for almost 7 years has gone.

For the same reason I think if we leave it there nobody will read that it's deprecated, thus, is about to be removed soon and shouldn't be used. 🤔

---

Newlines aren't supported and escaped with a space, "; " should work.

---

> If we remove `latest` it would read as it's already unaccessible. I can foresee users not finishing reading the description (thus, not seeing that it's been DEPRECATED but still available for a short period of time) and starting to ask why this tag that was in place for almost 7 years has gone.
> 
> For the same reason I think if we leave it there nobody will read that it's deprecated, thus, is about to be removed soon and shouldn't be used. 🤔

I think I would rather have people asking around when they failed to read rather than having people continuing to use the deprecated thing when they failed to read.

---

I hope developers will not remove it faster than we expect this to happen. My concern is that there is no other place where the set of values for this enum is specified. If this document were for users only, and the spec would still contain `latest` (marked as deprecated) then I would be less concerned.